.all-contributorsrc.json

```diff
@@ -0,0 +1,8 @@
+{
+  "files": ["readme.md"],
+  "imageSize": 100,
+  "contributorsPerLine": 7,
+  "contributorsSortAlphabetically": false,
+  "skipCi": "true",
```

Comment:
The default behavior is for the bot to skip the CI. I think this is what we want? We probably don't want to spend CPU cycles on adding the contributors?